CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Enable command arguments to change `{$Prefix}_EDITOR`. Updating IDiffOptions
+to include names for the files that are to be compared. Changes made to support recent zowe cli work: 
```

Comment:
For consumers reading the changelog, should we change "recent zowe CLI work" to a short phrase describing the work?